src/Request.php

```diff
@@ -112,16 +112,103 @@ private function before_execute() {
 		}
 	}
 
+	/**
+	 * Checks authentication errors.
+	 *
+	 * False will mean there are no detected errors and
+	 * execution will continue.
+	 *
+	 * Anything else (true, WP_Error, thrown exception, etc) will prevent execution of the GraphQL
+	 * request.
+	 *
+	 * @return boolean
```

Comment:
`@return` is the last option in the PHPDoc. The order should be `access`, `throws` and `return.`